test/legislature_test.rb

```diff
@@ -78,4 +78,46 @@ def test_csv_url_method
       assert_equal legislature.names_url, URI.join(base, sha + path).to_s
     end
   end
+
+  def test_upper_house_method
+    VCR.use_cassette('countries_json') do
+      bicameral = Everypolitician.country(code: 'CM')
+      assert_equal 'Sénat', bicameral.upper_house.name
+    end
+  end
+
+  def test_upper_house_with_unicameral_house
+    VCR.use_cassette('countries_json') do
+      unicameral = Everypolitician.country(code: 'GG-ALD')
+      assert_equal 'States', unicameral.upper_house.name
+    end
+  end
+
+  def test_lower_house_method
+    VCR.use_cassette('countries_json') do
+      bicameral = Everypolitician.country(code: 'CM')
+      assert_equal 'Assemblée Nationale', bicameral.lower_house.name
+    end
+  end
+
+  def test_lower_house_with_unicameral_house
+    VCR.use_cassette('countries_json') do
+      unicameral = Everypolitician.country(code: 'GG-ALD')
+      assert_equal 'States', unicameral.lower_house.name
+    end
+  end
+
+  def test_lower_house_with_multiple_matching_houses
+    VCR.use_cassette('countries_json') do
+      two_lower_houses = Everypolitician.country(code: 'VG')
```

Comment:
This, coupled with the next test, confused me greatly for a while, as if this country has two lower houses, then calling `.upper_house` on it should return `nil` rather than returning the House of Assembly.

What we're *really* testing here is the case where are two different houses in a unicameral legislature, so it would be much less confusing if the naming (of the test and the variable) reflected that! It's probably also worth having a test for `.upper_house` returning nil in the simple case of a country where we know there _is_ an upper house, but where we don't include it yet (e.g. the UK).